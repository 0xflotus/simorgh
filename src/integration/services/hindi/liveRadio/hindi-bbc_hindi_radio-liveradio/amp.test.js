/**
 * @service hindi
 * @pathname /hindi/bbc_hindi_radio/liveradio
 */

import runAmpTests from '../../../../pages/liveRadio/ampTests';

describe('AMP hindi liveRadio', runAmpTests);
