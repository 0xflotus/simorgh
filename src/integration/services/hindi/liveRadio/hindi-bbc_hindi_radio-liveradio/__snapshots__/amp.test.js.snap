// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP hindi liveRadio A11y I can see the skip to content link 1`] = `"सामग्री को स्किप करें"`;

exports[`AMP hindi liveRadio AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP hindi liveRadio AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP hindi liveRadio Analytics ATI 1`] = `
Object {
  "requests": Object {
    "base": "https://logws1363.ati-host.net?",
    "pageview": "\${base}s=598343&s2=52&p=hindi.bbc_hindi_radio.liveradio.page&r=\${screenWidth}x\${screenHeight}x\${screenColorDepth}&re=\${availableScreenWidth}x\${availableScreenHeight}&hl=\${timestamp}&lng=\${browserLanguage}&x1=[urn:bbc:ares:ws_media:live:bbc_hindi_radio]&x2=[amp]&x3=[news-hindi]&x4=[hi]&x5=[\${sourceUrl}]&x6=[\${documentReferrer}]&x7=[player-live]&x8=[simorgh]&x9=[बीबीसी+हिंदी+-+BBC+News+हिंदी]&ref=\${documentReferrer}",
  },
  "transport": Object {
    "beacon": false,
    "image": true,
    "xhrpost": false,
  },
  "triggers": Object {
    "trackPageview": Object {
      "on": "visible",
      "request": "pageview",
    },
  },
}
`;

exports[`AMP hindi liveRadio Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/hindi"`;

exports[`AMP hindi liveRadio Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. बाहरी साइटों की सामग्री के लिए बीबीसी ज़िम्मेदार नहीं है. बाहरी साइटों का लिंक देने की हमारी नीति के बारे में पढ़ें."`;

exports[`AMP hindi liveRadio Header I can see the branding 1`] = `"BBC News, हिंदी"`;

exports[`AMP hindi liveRadio I can see the headline 1`] = `"बीबीसी हिंदी"`;

exports[`AMP hindi liveRadio I can see the summary 1`] = `"दुनिया और भारत की ख़बरें, विश्लेषण और फ़ीचर. हिंदी में संवाद के सूत्रधार"`;

exports[`AMP hindi liveRadio Media player embed A11y I can read the media player title 1`] = `"ऑडिया प्लेयर"`;

exports[`AMP hindi liveRadio Media player embed I can see the media player embed 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/media/bbc_hindi_radio/liveradio/hi/amp?morph_env=live"`;

exports[`AMP hindi liveRadio SEO Canonical link 1`] = `"http://localhost:7080/hindi/bbc_hindi_radio/liveradio"`;

exports[`AMP hindi liveRadio SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP hindi liveRadio SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP hindi liveRadio SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP hindi liveRadio SEO Heading level 1 1`] = `"बीबीसी हिंदी"`;

exports[`AMP hindi liveRadio SEO Lang attribute 1`] = `"hi"`;

exports[`AMP hindi liveRadio SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "RadioChannel",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/hindi.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/hindi/bbc_hindi_radio/liveradio",
    "@type": "WebPage",
    "name": "बीबीसी हिंदी",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/hindi.png",
      "width": 1024,
    },
    "name": "BBC News हिंदी",
    "publishingPrinciples": "https://www.bbc.com/hindi/institutional-50223932",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/hindi.png",
  "url": "http://localhost:7080/hindi/bbc_hindi_radio/liveradio",
}
`;

exports[`AMP hindi liveRadio SEO OG description 1`] = `"दुनिया और भारत की ख़बरें, विश्लेषण और फ़ीचर. हिंदी में संवाद के सूत्रधार"`;

exports[`AMP hindi liveRadio SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/hindi.png"`;

exports[`AMP hindi liveRadio SEO OG image alt 1`] = `"BBC News हिंदी"`;

exports[`AMP hindi liveRadio SEO OG locale 1`] = `"hi-IN"`;

exports[`AMP hindi liveRadio SEO OG site name 1`] = `"BBC News हिंदी"`;

exports[`AMP hindi liveRadio SEO OG title 1`] = `"बीबीसी हिंदी - BBC News हिंदी"`;

exports[`AMP hindi liveRadio SEO OG type 1`] = `"website"`;

exports[`AMP hindi liveRadio SEO OG url 1`] = `"http://localhost:7080/hindi/bbc_hindi_radio/liveradio"`;

exports[`AMP hindi liveRadio SEO Page title 1`] = `"बीबीसी हिंदी - BBC News हिंदी"`;

exports[`AMP hindi liveRadio SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP hindi liveRadio SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP hindi liveRadio SEO Twitter creator 1`] = `"@bbchindi"`;

exports[`AMP hindi liveRadio SEO Twitter description 1`] = `"दुनिया और भारत की ख़बरें, विश्लेषण और फ़ीचर. हिंदी में संवाद के सूत्रधार"`;

exports[`AMP hindi liveRadio SEO Twitter image alt 1`] = `"BBC News हिंदी"`;

exports[`AMP hindi liveRadio SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/hindi.png"`;

exports[`AMP hindi liveRadio SEO Twitter site 1`] = `"@bbchindi"`;

exports[`AMP hindi liveRadio SEO Twitter title 1`] = `"बीबीसी हिंदी - BBC News हिंदी"`;
